cns/NetworkContainerContract.go

```diff
@@ -46,6 +51,12 @@ type CreateNetworkContainerRequest struct {
 	Routes                     []Route
 }
 
+// ConfigureNetworkContainerNetworkingRequest - specifies request to attach/detach network container to network.
```

Comment:
whats the difference between dockercontainerid and containerid? Is containerid not the docker container?

Reply:
containerid is from swift/dnc

Reply:
can we name this as swiftcontainerid then?